NetCoreTeamCity/Models/TestOccurrences.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace NetCoreTeamCity.Models
+﻿using System.Collections.Generic;
+
+namespace NetCoreTeamCity.Models
 {
     public class TestOccurrences
```

Comment:
@monkey3310 this model also use in BuildModel or we need to create new Model for List<TestOccurence>?